dll/win32/imm32/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 
-include_directories(${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine)
+include_directories(
+    ${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine
+    ${REACTOS_SOURCE_DIR}/win32ss/include)
 add_definitions(-D__WINESRC__)
 
 remove_definitions(-D_WIN32_WINNT=0x502)
```

Comment:
```suggestion

add_definitions(-D__WINESRC__)
remove_definitions(-D_WIN32_WINNT=0x502)
```

Reply:
by the way, is the `remove_definitions` one necessary?

Reply:
When removing `remove_definitions(-D_WIN32_WINNT=0x502)`:
```txt
<command-line>: error: "_WIN32_WINNT" redefined [-Werror]
```

Reply:
Both
`remove_definitions(-D_WIN32_WINNT=0x502)` and `add_definitions(-D_WIN32_WINNT=0x600)` removed.